lang/lang.pt-BR.json

```diff
@@ -206,50 +206,50 @@
                         ]
                     },
                     {
-                        "name": "Status",
+                        "name": "Estatísticas",
                         "value": [
                             "**Shards**: {{SHARD_COUNT}}",
-                            "**Servers**: {{SERVER_COUNT}} ({{SERVER_COUNT_PER_SHARD}}/Shard)"
+                            "**Servidores**: {{SERVER_COUNT}} ({{SERVER_COUNT_PER_SHARD}}/Shard)"
                         ]
                     },
                     {
                         "name": "Memória",
                         "value": [
                             "**RSS**: {{RSS_SIZE}} ({{RSS_SIZE_PER_SERVER}}/Server)",
-                            "**Heap**: {{HEAP_TOTAL_SIZE}} ({{HEAP_TOTAL_SIZE_PER_SERVER}}/Server)",
-                            "**Used**: {{HEAP_USED_SIZE}} ({{HEAP_USED_SIZE_PER_SERVER}}/Server)"
+                            "**Total**: {{HEAP_TOTAL_SIZE}} ({{HEAP_TOTAL_SIZE_PER_SERVER}}/Server)",
```

Comment:
["Heap" is a computer term](https://stackoverflow.com/questions/2308751/what-is-a-memory-heap), "Total" is too generic, I would leave this as "Heap".

Reply:
Done.